Swift Code Challenge/Views/Components/UsersListComponent.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  UsersListComponent.swift
+//  Swift Code Challenge
+//
+//  Created by Luiz Eduardo Mello dos Reis on 18/07/22.
+//
+
+import SwiftUI
+
+struct UsersListComponent: View {
+    @ObservedObject private var coordinator = UsersListView.Coordinator()
+    var body: some View {
+        List {
+            ForEach(self.coordinator.dataModel.people, id: \.self) { user in
+                NavigationLink(destination: UserDetailsView(user: user)) {
+                    HStack {
+                        AsyncImage(url: URL(string: user.picture.thumbnail ?? ""))
+                            .frame(width: 40, height: 40)
+                            .scaledToFill()
+                            .clipShape(Circle()).padding(.horizontal)
+                        Text("\(user.name.first ?? "Luiz") \(user.name.last ?? "Mello")")
+                    }
+                }
+                .navigationTitle("Users")
+            }
+            .onDelete(perform: removeRows)
+            .onMove(perform: move)
```

Comment:
Use autocomplete format to make clear. 

```suggestion
            .onMove(perform: move(from:,to:))
```